### ISSUE TITLE
Extend support to Fedora 30

### ISSUE THREAD

**Issue by buckaroogeek** [puppetlabs/bolt#986]
Fedora 30 was recently released by the Fedora community. May I please request that the build system be extended to include that release?

thank you

**Comment by nicklewis** [comment]
I've opened a [ticket in Jira](https://tickets.puppetlabs.com/browse/BOLT-1302) so we can prioritize this.

**Comment by donoghuc** [comment]
@buckaroogeek We just shipped a bolt package for fedora 30. You can install the repo with:
```
sudo rpm -Uvh https://yum.puppet.com/puppet6-release-fedora-30.noarch.rpm
sudo dnf install puppet-bolt
```

**Comment by buckaroogeek** [comment]
@donoghuc  Wonderful!

Thank you. As a minor comment, I was using bolt from the fedora 29 repo, so after adding the fedora 30 rpm, `sudo dnf update` was enough to pull the updated bolt package.

Brad

**Comment by donoghuc** [comment]
Fedora 30 packages available starting with version 1.24.0.
